product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/index.mdx

```diff
@@ -20,6 +20,8 @@ With BigAnimal, you have a few options for monitoring and logging solutions:
 
 - BigAnimal provides a Prometheus-compatible endpoint you can use to connect to your own metrics infrastructure, such as your AWS Managed Grafana. It also provides the option to view logs from your cloud provider's blob storage solution. For more information, see [Other monitoring and logging solutions](other_monitoring). This ability is an optional solution when you're using your own cloud account but is the only option when using BigAnimal's cloud account.
 
+- BigAnimal provides another Prometheus-compatible endpoint PEMx, that you can use to consolidate and expose metrics in each data-plane. PEMx is an integrated monitoring and alerting solution within BigAnimal. It is specially designed to monitor performance of the Postgres clusters. See [PEMx monitoring](monitoring_using_pemx) for more information.
```

Comment:
`BigAnimal provides another Prometheus-compatible endpoint PEMx, that you can use to consolidate and expose metrics in each data-plane.` This line doesn't seem technically accurate to me. wdyt @xmarcelet ?

Reply:
"BigAnimal provides another Prometheus-compatible endpoint PEMx, that you can use to consolidate and expose metrics in each data-plane," I don't think we need to add this line.

Reply:
My interpretation is that:
- The text above `BigAnimal provides a Prometheus-compatible endpoint you can use to connect to your own metrics infrastructure [...]` refers to the Thanos endpoint that exposes the **infrastructure metrics** that used by PEMx to generate **PEM metrics** (the ones displayed in the `Monitoring & Logging` page).
- This text should instead refer to the option to view the **PEM metrics** directly through the `Monitoring & Logging`.

Reply:
PEMx does not provide any prometheus-compatible endpoint.
A more accurate sentence would be: `BigAnimal provides another metric endpoint: PEMx`

Reply:
changes done